nodestream_akamai/edns/edns.py

```diff
@@ -0,0 +1,30 @@
+import json
+import logging
+
+from nodestream.pipeline.extractors import Extractor
+
+from ..akamai_utils.edns_client import AkamaiEdnsClient
+
+
+class AkamaiEdnsExtractor(Extractor):
+    def __init__(self, **akamai_client_kwargs) -> None:
+        self.client = AkamaiEdnsClient(**akamai_client_kwargs)
+        self.logger = logging.getLogger(self.__class__.__name__)
+
+    async def extract_records(self):
+        for zone in self.client.list_zones():
```

Comment:
@zprobst I am finding myself wanting a sample json file for testing so I can see the data that is being processed.  I think this is a manifestation of not having proper tests in this plugin.  

@stuartio is there a way to include a sample json test file in this repo somehow in preparation for doing some testing?  Removing any specific data from Intuit and making it mock data of course...

Reply:
Have added NS and CAA records. Scoped out SOA